docs/version0.30.md

```diff
@@ -30,7 +30,7 @@ The following new features and notable changes since version 0.29.0 are included
 - [Changes to the shared classes cache generation number](#changes-to-the-shared-classes-cache-generation-number)
 - [Ignored options now captured in java dumps](#ignored-options-captured-in-java-dumps)
 - [New `-XX:[+|-]EnsureHashed` option added](#new-xx-ensurehashed-option-added)
-- [Redesigned heap resizing for balanced GC policy](#redesigned-heap-resizing-for-balanced-gc-policy)
+- [Redesigned heap resizing for the the `balanced` GC policy](#redesigned-heap-resizing-for-the-balanced-gc-policy)
```

Comment:
This contains "the the"

Reply:
@pshipton - I have corrected the document based on this comment. Can I go ahead and commit the changes so that it can be visible in the stage or should I expect some review comments from @amicic? Thanks!

Reply:
Pls commit the changes and squash what you've got. I'm not expected comments from Aleks in the short term, I would go ahead and merge this and Aleks can request changes later if necessary.

Reply:
@pshipton - Committed the changes and squashed the commits as advised. Thanks!